### ISSUE TITLE
RSS

### ISSUE THREAD

**Issue by raphaelbastide** [EtienneOz/UrlDropper#8]
RSS feed would be handy!
Shaarly provides it by default e.g: http://sebsauvage.net/links/?do=rss

**Comment by EtienneOz** [comment]
You're right. Actually, rss feeds are already located here http://www.etienneozeray.fr/shaarli/?do=rss&searchtags=DropIt. I just have to put a link on the site.

**Comment by raphaelbastide** [comment]
Great!
Followed.
